### PR TITLE
Bug 2037203: Add a scrollable legend to Running VMs graph

### PR DESCRIPTION
**Pull request by pcbailey**
The default legend for PatterFly charts are SVG elements that aren't scrollable. This PR replaces that legend with an HTML element that allows for scrolling.

Screenshots:

Before:
Selection_047

After:
Selection_046

### CODE REVIEW

**Comment by pcbailey** [comment]
@yaacov @avivtur @metalice @glekner @vojtechszocs @hstastna @upalatucci PTAL

File: frontend/packages/kubevirt-plugin/src/components/virtualization-overview/running-vms-per-template-card/VMsChartLegendLabel.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+import './running-vms-per-template-card.scss';
+
+export const VMsChartLegendLabel = ({ item }) => {
+  const iconStyle = { color: item.color };
+
+  return (
+    <>
+      <i className="fas fa-square kv-running-vms-card__legend-label--color" style={iconStyle} />
+      <span className="kv-running-vms-card__legend-label--count">{item.vmCount}</span> {item.name}
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/src/components/virtualization-overview/running-vms-per-template-card/VMsChartLegendLabel.tsx:11]: notes for improvments:
a - `item` can have a type.
b - You can add link:
`/k8s/ns/<NS>/kubevirt.io~v1~VirtualMachine?labels=vm.kubevirt.io/template=<template name>`

for example:
`/k8s/all-namespaces/kubevirt.io~v1~VirtualMachine?labels=vm.kubevirt.io/Ftemplate=windows10-desktop-medium`

Comment (pcbailey) [reply frontend/packages/kubevirt-plugin/src/components/virtualization-overview/running-vms-per-template-card/VMsChartLegendLabel.tsx]: I didn't see that you had lgtm'd it. I made the suggested changes. PTAL

**Pull request merged**